arches_lingo/src/arches_lingo/components/generic/ResourceInstanceRelationshipsEditor.vue

```diff
@@ -0,0 +1,34 @@
+<script setup lang="ts">
+import { computed, toRefs } from "vue";
+import MultiSelect from "primevue/multiselect";
+import type { ResourceInstanceReference } from "@/arches_lingo/types";
+
+const props = defineProps<{
+    val?: string[];
+    options?: ResourceInstanceReference[];
+}>();
+
+const emit = defineEmits(["update"]);
+
+const propRefs = toRefs(props);
+
+const value = computed({
+    get() {
+        return propRefs.val.value;
+    },
+    set(value) {
+        emit("update", value);
+    },
+});
+</script>
+<template>
+    <div>
+        <MultiSelect
+            v-model="value"
+            :options="props.options"
```

Comment:
The checkbox to select all is a little confusing when there are no options, suggest
```suggestion
            :show-toggle-all="!!props.options?.length"
            :options="props.options"
```